lib/pxconfig.js

```diff
@@ -41,6 +45,29 @@ class PxConfig {
         return Object.assign(this.PX_DEFAULT, this.PX_INTERNAL);
     }
 
+    mergeConfigFileParams(configParams) {
+        let mergedParams = configParams;
+        try {
+            let fileConfig = {};
+            const filepath = configParams.configFilePath;
+            if (filepath) {
+                fileConfig = pxutil.readJsonFile(filepath);
+            }
+            const fileMappedConfig = {};
+            Object.keys(fileConfig).forEach(key => {
```

Comment:
why not use `for (let key in fileConfig)`?

Reply:
changed (works the same)